pkg/comp-functions/functions/vshnminio/minio_deploy.go

```diff
@@ -295,3 +307,27 @@ func createServiceMonitor(ctx context.Context, comp *vshnv1.VSHNMinio, iof *runt
 
 	return iof.Desired.PutIntoObject(ctx, sm, comp.Name+"-service-monitor")
 }
+
+func createSliBucket(ctx context.Context, comp *vshnv1.VSHNMinio, xminioName string, iof *runtime.Runtime) error {
+	obj := &v1.ObjectBucket{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      SLIBucketName,
+			Namespace: comp.GetInstanceNamespace(),
+		},
+		Spec: v1.ObjectBucketSpec{
+			Parameters: v1.ObjectBucketParameters{
+				BucketName: SLIBucketName,
+				Region:     "rma",
```

Comment:
I wouldn't use a cloudscale.ch region here. Please use Minio's default region `us-east-1`.

Currently it doesn't matter, as we don't enable region support in Minio, but if we ever would, it probably breaks with this region.